freeplane/src/main/java/org/freeplane/view/swing/map/MapViewPane.java

```diff
@@ -1,613 +1,38 @@
 package org.freeplane.view.swing.map;
 
 import java.awt.BorderLayout;
-import java.awt.Color;
-import java.awt.Component;
 import java.awt.Container;
-import java.awt.Cursor;
-import java.awt.Graphics;
-import java.awt.Graphics2D;
 import java.awt.Insets;
-import java.awt.LayoutManager;
-import java.awt.Point;
-import java.awt.Rectangle;
-import java.awt.RenderingHints;
 import java.awt.event.ComponentAdapter;
 import java.awt.event.ComponentEvent;
-import java.awt.event.MouseEvent;
-import java.awt.event.MouseWheelEvent;
-import java.awt.geom.AffineTransform;
-import java.awt.image.BufferedImage;
-import java.util.EnumSet;
-import java.util.Optional;
 
-import javax.swing.BoundedRangeModel;
-import javax.swing.JComponent;
 import javax.swing.JLayeredPane;
 import javax.swing.JPanel;
-import javax.swing.JScrollBar;
 import javax.swing.JScrollPane;
-import javax.swing.JViewport;
-import javax.swing.SwingConstants;
 import javax.swing.SwingUtilities;
-import javax.swing.border.Border;
-import javax.swing.event.MouseInputAdapter;
 import javax.swing.event.MouseInputListener;
 
 import org.freeplane.core.resources.IFreeplanePropertyListener;
 import org.freeplane.core.resources.ResourceController;
-import org.freeplane.core.ui.components.resizer.UIComponentVisibilityDispatcher;
-import org.freeplane.core.util.Compat;
 import org.freeplane.features.map.IMapChangeListener;
 import org.freeplane.features.map.MapChangeEvent;
 import org.freeplane.features.mode.Controller;
-import org.freeplane.features.ui.IMapViewManager;
 import org.freeplane.features.ui.ViewController;
+import org.freeplane.view.swing.map.overview.MapOverviewImage;
```

Comment:
Could you move also MapViewPane to the new package so that most classes of the new package do not need to be public, visible, and known outside the package?
By the way, in the best of all possible worlds, the first commit would only extract the classes and the second only move all classes to the new package, it was nice but it is not really important.

Reply:
Will be treated in next commits.